_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,206 @@
+
+### Maintaining other ecosystem plugins: like JSX/Flow.
+
+There is both a React/Flow team at Facebook and they have many issues/PRs associated with Babel that we have to review/maintain.
+
+- [React plugins](https://babeljs.io/docs/plugins/#react)
+- [Flow plugin](https://babeljs.io/docs/plugins/transform-flow-strip-types)
+
+### Autoprefixer for Babel via [babel-preset-env](https://github.com/babel/babel-preset-env)
+
+This is something that is very important as we move forward to a point where users are compiling a specific set of transforms while the browsers support a specific set of features. There is a constant sliding window. This preset picks the right amount of plugins supported by the target environments so that the code can be fast/smaller.
+
+The next big feature would be dynamic checking of polyfills as well as syntax as described in [babel/babel-preset-env#20](https://github.com/babel/babel-preset-env/issues/20).
+
+### Linting via [babel-eslint](https://github.com/babel/babel-eslint)
+
+As long as you want to lint 
+with experimental features (stage-x, types) that ESLint proper won't support yet.
+
+Future: Better interop with ESLint (hook into scope, traversal), possibility of writing an ESLint rule with Babel APIs, running ESLint rules as plugins/presets?
+
+### Minification
+
+[babel/babili](https://github.com/babel/babili) was created as a future proof minifier.
+
+It will be continually updated to take advantage of the latest JavaScript features (ES2015+ aware). Each transform can be independent and also pluggable.
+
+### Codemods/Refactoring
+
+[jscodeshift](https://github.com/facebook/jscodeshift), [js-codemod](https://github.com/cpojer/js-codemod), [Lebab](https://github.com/lebab/lebab).
+
+By passing in recast in the options, Babel can also be run on source code and can power the future of codemods. This feature was just made possible so looking forward to making it easier to do and the transformations it can enable.
+
+### Code Coverage / Instrumentation
+
+With tools like [nyc](https://github.com/istanbuljs/nyc) and [babel-plugin-instanbul](https://github.com/istanbuljs/babel-plugin-istanbul).
+
+### API Documentation / Community
+
+Meetups, tutorials, videos, open source workshops, etc.
+Docs should be explaining each feature and possible usages, improved links to books, articles, etc.
+
+- All plugins should have examples/repl links. Can also embed runkit.
+- Update Learn ES2015 to have ES2015+.
+- Add resources page
+- Update FAQ with common errors
+- Page for community plugins
+- babel-handbook, API docs, etc
+
+## Future
+
+### [Plugin API Changes](https://github.com/babel/notes/blob/master/2016-08/august-01.md#potential-api-changes-for-traversal)
+
+Currently many users have trouble with plugin/preset ordering and it's very confusing what plugin is run before/after and how to make it do so. This is a fundamental change to the core of Babel which will take some work to figure out.
```

Comment:
Not sure if I'm capturing the essence of what you're trying to say: "Currently many users have trouble with plugin/preset sequencing (i.e., how to make a plugin run before/after another plugin)"